sources/packages/api/src-sql/Types/Remove-scholastic-standing-status-enum-and-col.sql

```diff
@@ -0,0 +1,6 @@
+-- Remove column scholastic_standing_status for student_scholastic_standings.
+ALTER TABLE
```

Comment:
Is there a reason to create a second migration file to remove more columns from the same `sims.student_scholastic_standings` table?

Reply:
I did it to group the enum and dependent column, ya... i can them all in a single file